### ISSUE TITLE
Discovery: Accessibility Report for issues within the CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10273]
## Background
In order to ensure we are making improvements to accessibility within the VA.gov CMS, we need to establish a consistent approach to finding and reporting issues.

### Possible scanning tools
- Axe-core
- Purple Hats
- Pa11y
- Screaming Frog

## Additional Context
This work will support ATAG 2.0 Part A: Make the authoring tool user interface accessible

## Acceptance Criteria
- [x] Research & analyze various scanning/reporting tools
- [x] Determine which one might work best for our needs (technical, usability and accessibility)
- [x] Determine recurring/consistent schedule to run the report

**Comment by EWashb** [comment]
@laflannery this one has been added to Sprint 65. Feel free to point!

**Comment by ndouglas** [comment]
Related to/overlaps with #9921.

**Comment by laflannery** [comment]
Notes from meeting with Nate:
- Current scanning is being done with cypress-axe
  - Implemented in January 2021
  - Using WCAG 2.0 A and AA standards
  - Scans the logged out Homepage and most "add" content pages
  - Runs on Pull requests and on staging to any change to the main branch
    - PR cannot be approved or merged until the tests are passing
  - Not many issues come up, this could be because not many pages are being tested

Next Steps:
- Increase pages being scanned
- Update standards used to WCAG 2.1 (wcag21a, wcag21aa) and also include Best Practices (best-practice)

Future steps:
- Currently reporting of errors is a nightmare.  They are difficult to understand and unclear. Reports are also not permanent, meaning they cannot be used to track progress over time.
- Ideally, we would like to have clear, easy to understand reporting of errors that can be used over time to show that the type and number of errors within the CMS is going down.
- It was also discusses that if there is a different tool that could do this better than cypress-axe does currently, that wouldn't be out of the question

**Comment by ndouglas** [comment]
Related to above:

- #9921
- #10364
- #10368 
- #10369
- #10370
- #10371
